sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIService.scala

```diff
@@ -73,15 +73,15 @@ private[hive] class SparkSQLCLIService(hiveContext: HiveContext)
 }
 
 private[thriftserver] trait ReflectedCompositeService { this: AbstractService =>
-  def initCompositeService(hiveConf: HiveConf) {
+  def initCompositeService(hiveConf: HiveConf, level: Int = 2) {
```

Comment:
The `level` argument looks really evil, but I've no idea how to get rid of it right now...